powermon/protocols/pi18.py

```
@@ -112,9 +744,13 @@ def check_crc(self, response: str, command_definition: CommandDefinition = None)
             else:
                 log.info("PI18 response check_crc doesnt match got (%x, %x), calc (%x, %x)", crc_high, crc_low, response[-3], response[-2])
                 return False
```

should `raise InvalidCRC(message)` (see PI30)